posts/2021-05-07-caught-red-handed.md

```diff
@@ -22,44 +24,51 @@ from the appropriate page on our website.
 
 [install]: https://www.rust-lang.org/install.html
 
-# What's in 1.52.1 stable
-
-This point release contains a single change: it disables incremental
-compilation. Read on for more details as to why, and the next steps the Rust
-project is pursuing on this issue.
-
 # Summary
 
-The Rust teams are always excited to report on new features offered with each release. Sometimes, however, an important change that is not yet "fully baked" gets accidentally included in a release, and we need to issue a point release.
-
-There was an instance of this in the most recent release, 1.52.0, which added a new bit of internal-consistency checking, called "incremental compilation hash verification" (abbreviated `verify-ich`). This check is also called an "unstable fingerprint" check, because the diagnostic it currently prints look [like this](https://github.com/rust-lang/rust/issues/84336):
-
-```
-thread 'rustc' panicked at 'assertion failed: `(left == right)`
-  left: `Some(Fingerprint(4565771098143344972, 7869445775526300234))`,
-   right: `Some(Fingerprint(14934403843752251060, 623484215826468126))`: found unstable fingerprints for <massive text describing rustc internals elided>
-
-error: internal compiler error: unexpected panic
+This release works around broken builds on 1.52.0, which are caused by newly
+added verification. The bugs this verification detects are present in all Rust
+versions, and can trigger miscompilations in incremental builds, so downgrading
```

Comment:
I would say "all Rust versions since 1.24.0, when incremental compilation was turned on by default", according to here: https://github.com/rust-lang/rust/blob/master/RELEASES.md#cargo-27, I can't find quickly where incremental compilation was first shipped in stable. Effectively, this is "all Rust versions" because people usually upgrade, but I think it helps to be more specific if we can.

Reply:
Rust 1.24 is correct - I'm going to add this as a footnote, so we're not distracting too much from the central message. It was all the way back in February 2018, so my guess is most users have migrated.